### PR TITLE
In particular, forgetting SSH_AGENT_KEY leads to inscrutable errors

### PR DESCRIPTION
**Pull request by jibsheet**
This wil make it clearer what config you forgot to set up in EXTRA_VARS

@edx/devops

### CODE REVIEW

**Comment by jibsheet** [comment]
@estute FYI - I started using assert() and I found the : very helpful (gives a better error message, a bare assert was generating a ginormous stack trace for me which is harder for folks to parse)

File: devops/jobs/RunAnsible.groovy [hunk 1]
```diff
@@ -37,11 +37,14 @@ import static org.edx.jenkins.dsl.DevopsConstants.common_read_permissions
 class RunAnsible {
     public static job(DslFactory dslFactory, String jobName, String environment, String deployment, Map extraVars) {
         return dslFactory.job(extraVars.get("FOLDER_NAME") + "/${environment}-${deployment}-${jobName}") {
+            // These 3 don't have defaults
+            assert extraVars.containsKey('SSH_AGENT_KEY') : "Please defined SSH_AGENT_KEY"
```
Comment (estute) [inline devops/jobs/RunAnsible.groovy]: nit: typo

**Comment by estute** [comment]
@jibsheet may/may not make sense to assert on all necessary variables. This can be helpful, especially if any of them are stored in secrets and therefore difficult to determine their values

**Comment by jibsheet** [comment]
@estute Yeah, these are the 3 for this job.  The rest have sensible defaults in the .get('var','default').
I plan to sprinkle some more in the seed jobs for similar reasons, but started here.

**Pull request merged**